packages/mobile/src/components/details-tile/DetailsTileActionButtons.tsx

```diff
@@ -54,19 +61,25 @@ const useStyles = makeStyles(({ palette }) => ({
 export const DetailsTileActionButtons = ({
   hasReposted,
   hasSaved,
+  isPlaylist = false,
```

Comment:
since isPlaylist is set in the parent component, we dont need the default value here. typically we should avoid all default args all the way down, and just make sure it's set in one spot, like the highest parent?